packages/connect/src/data/DataManager.ts

```diff
@@ -54,18 +26,35 @@ export class DataManager {
     private static messages: Record<string, any>;
 
     static async load(settings: ConnectSettings, withAssets = true) {
-        const ts = settings.env === 'web' ? `?r=${settings.timestamp}` : '';
         this.settings = settings;
 
         if (!withAssets) return;
 
-        const assetPromises = assets.map(async asset => {
-            const json = await httpRequest(`${asset.url}${ts}`, 'json');
-            this.assets[asset.name] = json;
-        });
-        await Promise.all(assetPromises);
+        const coins = require('@trezor/connect-common/files/coins.json');
+        const coinsEth = require('@trezor/connect-common/files/coins-eth.json');
+        const bridge = require('@trezor/connect-common/files/bridge/releases.json');
+        const t1b1 = require('@trezor/connect-common/files/firmware/t1b1/releases.json');
```

Comment:
I am still seeing a fetch for releases.json, its probably because of some firmware security check. something to discuss
<img width="529" alt="image" src="https://github.com/user-attachments/assets/e318a1eb-c82d-4771-9fe2-e3800ae6eb2a">

<img width="531" alt="image" src="https://github.com/user-attachments/assets/8a25bdc6-6b70-4f4f-86ef-05407bca3db4">

Reply:
Yes, that is probably an optimization that can be done. Interesting.

Reply:
Yes and no.
What we really need is only to have _up to date_ version of `releases.json`.
- On suite **desktop** that does mean we need to fetch `data.trezor.io`, because suite might be outdated
- But suite **web** is always up to date, so yes, `releases.json` can be included in the bundle :heavy_check_mark: 
  - theoretically you could have the same browser tab open for months, but that is a negligible edge case IMO